src/app/component/component.js

```diff
@@ -0,0 +1,8 @@
+import template from './component.html';
+import './component.scss';
+
+const AppComponent = {
+    template
+};
+
+export default AppComponent;
```

Comment:
I think that our way of creating component is shorter and more readable. You may try it
![image](https://user-images.githubusercontent.com/37654233/57124042-69d22080-6d84-11e9-91e2-e0d25e3351eb.png)